### DISCUSSION TITLE
Improving Sloth SLOs dashboard

### DISCUSSION THREAD

**Discussion by w-reichert** [grafana/grafana#41363] [Dashboards, Panels & Visualizations]
First of all, many thanks for the Sloth SLOs sample dashboard (https://grafana.com/grafana/dashboards/14348)! We are using it since a while. :-) 

I noticed that the color coding and ranges for" Remaining error budget (month)" is not correct. 
It starts in red if there are no values, it is yellow if there are no errors, and it is green if the budget is below 40%.
Furthermore, I suppose it should be cut off negative values since empty is empty.

My suggestions:

```
          "description": "This month remaining error budget, starts the 1st of the month and ends  28th-31st (not rolling window)",
          "fieldConfig": {
            "defaults": {
              "color": {
                "mode": "thresholds"
              },
              "mappings": [],
              "max": 1,
              "min": 0,
              "thresholds": {
                "mode": "absolute",
                "steps": [
                  {
                    "color": "grey",
                    "value": null
                  },
                  {
                    "color": "red",
                    "value": 0
                  },
                  {
                    "color": "orange",
                    "value": 0.01
                  },
                  {
                    "color": "light-yellow",
                    "value": 0.2
                  },
                  {
                    "color": "green",
                    "value": 0.8
                  }
                ]
              },
              "unit": "percentunit"
            },
            "overrides": []
          },
```
and alike for ` "description": "A rolling window of the total period (30d) error budget remaining."`

Cutting of negative budget (also occurs twice).
```
              "expr": "1-clamp_max(sum_over_time( ... ) , 1)",
```

Thanks and regards, Wolfgang

**Comment by w-reichert** [comment]
Issue moved to https://github.com/slok/sloth/issues/196

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
